packages/thumbprint-react/components/Alert/test.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { BannerAlert, InPageAlert } from './index';
+
+describe('BannerAlert', () => {
+    test('renders messageType good', () => {
+        const wrapper = mount(
+            <BannerAlert messageType="good">
+                <p>Alert content</p>
+            </BannerAlert>,
+        );
+        expect(
+            wrapper
+                .find('div')
+                .at(0)
+                .hasClass('goodNews'),
+        ).toBe(true);
+        expect(wrapper).toMatchSnapshot();
+    });
+    test('renders messageType bad', () => {
+        const wrapper = mount(
+            <BannerAlert messageType="bad">
+                <p>Alert content</p>
+            </BannerAlert>,
+        );
+        expect(
+            wrapper
+                .find('div')
+                .at(0)
+                .hasClass('badNews'),
+        ).toBe(true);
+        expect(wrapper).toMatchSnapshot();
+    });
+    test('renders messageType warning', () => {
+        const wrapper = mount(
+            <BannerAlert messageType="warning">
+                <p>Alert content</p>
+            </BannerAlert>,
+        );
+        expect(
+            wrapper
+                .find('div')
+                .at(0)
+                .hasClass('warning'),
+        ).toBe(true);
+        expect(wrapper).toMatchSnapshot();
+    });
+    test('renders messageType note', () => {
+        const wrapper = mount(
+            <BannerAlert messageType="note">
+                <p>Alert content</p>
+            </BannerAlert>,
+        );
+        expect(
+            wrapper
+                .find('div')
+                .at(0)
+                .hasClass('note'),
+        ).toBe(true);
+        expect(wrapper).toMatchSnapshot();
+    });
+    test('renders children', () => {
+        const wrapper = mount(
+            <BannerAlert messageType="note">
+                <p>Alert content</p>
+            </BannerAlert>,
+        );
+        expect(
+            wrapper
+                .find('div')
+                .at(1)
+                .text(),
+        ).toBe('Alert content');
```

Comment:
I recommend re-writing adding a `data-test-id="text"` attribute to the `p` element and then writing the test as `wrapper.find('[data-test-id="text"]).text()` rather than doing the `find('div')` and `at(1)`. This will make the test more resilient to changes in the component markup.

Reply:
On it! Updated in this commit https://github.com/thumbtack/thumbprint/pull/521/commits/0737da6df1b651e9fd2b8349778f5ea432ddad10